### PR TITLE
Add a JOSS paper draft for submission

### PR DESCRIPTION
**Pull request by waxlamp**
Hi all-

This branch contains the materials for a submission to JOSS. You can look at the [submission guidelines](https://joss.readthedocs.io/en/latest/submitting.html) and take a look at [this paper](https://joss.theoj.org/papers/10.21105/joss.02105) that I recently got accepted to see an example of what these are supposed to look like.

### CODE REVIEW

**Review by subdavis — APPROVED** [review]

File: joss/paper.md [hunk 1]
```diff
@@ -0,0 +1,417 @@
...
+Similarly, the sum of all values for each sample can be scaled to a set value.
+The default value in Viime is 100.  Other options include normalization based on
+a column containing sample weights or volumes.  The next step is data
+transformation.  Often times, data is transformed to bring the distribution
+closer to normality and to compress the dynamic range.  The options in this step
+are  Log10, Log2, square root and cube root.
+
+Lastly, the data can be scaled.  This also addresses the issue of large dynamic
+range by scaling the variance of the data.  The options here include
+Autoscaling, Pareto scaling, Range scaling, Vast scaling and Level scaling.
```
[the remark this answers is not in the document]
Comment (waxlamp) [reply joss/paper.md:159]: "Pareto" is a name so needs to remain capitalized, but you are right about the other terms.

File: joss/paper.md [hunk 2]
```diff
@@ -0,0 +1,417 @@
+---
+title: 'Viime: Visualization and Integration of Metabolomics Experiments'
```
[the remark this answers is not in the document]
Comment (waxlamp) [reply joss/paper.md:2]: See 2e33b0007fd2c37d9d8f02bb352cc28c7296dd70.

File: joss/paper.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (waxlamp) [reply joss/paper.md]: See 12585ec440b1d7b5c6db51521c5fe8c38ae0c68c.

**Review by subdavis — APPROVED** [review]
:100:

**Pull request merged**